### PR TITLE
Fixes commonshost/dohnut#20 and adds docker-compose documentation example

### PR DESCRIPTION
**Pull request by wknd**
I fixed the issue by adding bootstrap to the list of parameters that gets split into an array in cli.js
and added documentation on how to use dohnut with docker-compose
additionally I added node_modules and package-lock.json to .gitignore since those files weren't in the repo before and would be created if testing locally

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 !.travis.yml
 !.gitignore
 qemu-*
+node_modules/*
```
Comment (sebdeckers) [inline .gitignore]: Please leave these unrelated changes out of this PR.

**Comment by wknd** [comment]
I also bumped the version so it could instantly be used since its such a small fix.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dohnut",
-  "version": "4.6.0",
+  "version": "4.6.1",
```
Comment (sebdeckers) [inline package.json]: You can leave the version bump out of the PR. I'll do that separately when any other changes (if any) at time of `npm publish`.

File: docs/docker/README.md [hunk 3]
```diff
@@ -18,6 +18,23 @@ Run forever as a background service, listen on port `53/udp` on all network inte
...
+version: "3"
+
+services:
+  dohnut:
+    container_name: dohnut
+    image: commonshost/dohnut:latest
+    environment:
+      TZ: "Europe/Brussels"
+      DOHNUT_LISTEN: 127.0.0.1:53
+      DOHNUT_BOOTSTRAP: 1.1.1.1
```
Comment (sebdeckers) [inline docs/docker/README.md:32]: Did you mean to add multiple IPs here?

**Review by sebdeckers — CHANGES_REQUESTED** [review]
Good find & thank you for the fix. Just a few nits.

File: package.json (hunk 2 above)
Comment (sebdeckers) [reply package.json]: I think this would still be a minor version bump since some behaviour changes in a (hopefully) backwards compatible manner.

File: docs/docker/README.md (hunk 3 above)
Comment (wknd) [reply docs/docker/README.md:32]: In the documentation I wanted to follow the above example which only used 1 ip. But you definitely can use multiple ip's when the fix is applied.

File: package.json (hunk 2 above)
Comment (wknd) [reply package.json]: I don't think it should break anything existing, but I'll leave that to you. I took out the version bump in the next commit.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dohnut",
-  "version": "4.6.0",
+  "version": "4.5.3",
```
Comment (sebdeckers) [inline package.json]: 🤔

**Review by sebdeckers — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (wknd) [reply package.json]: wait wut

Comment (wknd) [reply package.json]: did I just checkout the wrong hash? . . .

Comment (sebdeckers) [reply package.json]: And that's why I am an `commit --amend` & `push --force` addict. 😅

**Comment by sebdeckers** [comment]
Thank you @wknd 🥰

FWIW my `~/gitignore_global` is:

```
*~
.DS_Store
/node_modules/
package-lock.json
/venv/
/.vscode/
```

**Comment by sebdeckers** [comment]
Bumped & published to NPM. The bump should get picked up by Travis CI and published to Docker Hub momentarily.

**Comment by wknd** [comment]
Awesome! Time to continue setting up my raspberry pi.
